src/bin/commands/new.js

```diff
@@ -98,6 +98,7 @@ function createModuleProject(args, name, targetDir, cb) {
     let devDependencies = ['react', 'react-dom', 'create-react-styleguide'];
     if (args.eslint === 'zillow') {
         devDependencies.push('eslint-plugin-zillow');
+        devDependencies.push('eslint-plugin-jest');
```

Comment:
`devDependencies.push('eslint-plugin-zillow', 'eslint-plugin-jest');`

Reply:
You probably also need husky and jest-emotion here, otherwise the dependencies will not be installed by simply adding them to the package.json template.